{"name": "Создание заказа при неверном хеше ингредиентов", "status": "passed", "start": 1701267213423, "stop": 1701267213652, "uuid": "b688c304-4076-4b64-bd2c-03184fd352b0", "historyId": "1ef5c72fd23a8885ac873ba0deb08da9", "testCaseId": "1ef5c72fd23a8885ac873ba0deb08da9", "fullName": "tests.test_create_order.TestCreateOrder#test_create_order_wrong_hash_fail", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_order"}, {"name": "subSuite", "value": "TestCreateOrder"}, {"name": "host", "value": "i101922157"}, {"name": "thread", "value": "12744-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_order"}]}